packages/cardano-services/src/ChainHistory/DbSyncChainHistory/ChainHistoryBuilder.ts

```diff
@@ -92,7 +92,7 @@ const getGovernanceAction = ({
   numerator,
   type
 }: ProposalProcedureModel): Cardano.GovernanceAction => {
-  const { contents } = JSON.parse(description);
+  const { contents } = description;
```

Comment:
This doesn't seem to be validated with openapi schema. It would be good to check the shape of this object to catch db-sync schema changes

Reply:
LW-10179 should solve this